test/functional/sqlsrv/sqlsrv_ae_output_param_sqltype_decimal_precision.phpt

```diff
@@ -0,0 +1,264 @@
+                                if (AE\isDataEncrypted()) {
+                                    if ($r !== false) {
+                                        echo "AE: Conversion from $typeFull to output $sqltypeFull should not be supported\n";
+                                    } else {
+                                        if (sqlsrv_errors()[0]['SQLSTATE'] != "22018") {
+                                            echo "AE: Conversion from $typeFull to output $sqltypeFull expects an operand type clash error, actual error is incorrect\n";
+                                            var_dump(sqlsrv_errors());
+                                        }
+                                    }
+                                } else {
+                                    if ($n1 - $n2 < $m1 - $m2) {
+                                        if ($r !== false) {
+                                            echo "Conversion from $typeFull to output $sqltypeFull should not be supported\n";
+                                        }
+                                    } else {
+                                        if ($r === false) {
+                                            echo "Conversion from $typeFull to output $sqltypeFull should be supported\n";
+                                        } else {
+                                            if (abs($c_detOut - $inputValues[0]) > $epsilon || abs($c_randOut - $inputValues[1]) > $epsilon) {
+                                                echo "Conversion from $typeFull to output $sqltypeFull causes data corruption\n";
+                                            }
+                                        }
+                                    }
+                                }
+                            // check the case when the SQLSRV_SQLTYPE precision (n1) and scale (n2) are the same as the column precision (m1) and scale (m2)
+                            // should work with AE or non AE
+                            } else {
+                                if ($r === false) {
+                                    echo "Conversion from $typeFull to output $sqltypeFull should be supported\n";
+                                } else {
+                                    if (abs($c_detOut - $inputValues[0]) < $epsilon && abs($c_randOut - $inputValues[1]) < $epsilon) {
+                                        echo "****Conversion from $typeFull to output $sqltypeFull is supported****\n";
+                                    } else {
+                                        echo "Conversion from $typeFull to output $sqltypeFull causes data corruption\n";
+                                    }
+                                }
+                            }
+                            // cleanup
+                            sqlsrv_free_stmt($stmt);
+                        }
+                    }
+                }
+            }
+            dropTable($conn, $tbname);
```

Comment:
droProc - it's good to cleanup after yourself even tho next test will always drop before it creates